frontend/model/contracts/group.js

```diff
@@ -1166,12 +1166,19 @@ sbp('chelonia/defineContract', {
         if (rootState[data.chatRoomID]?._volatile) {
           return
         }
-
+        // the specific scenario here is joining chatrooms (should be the general channel in this
```

Comment:
The comments here are fine. It'll cause a merge conflict with my changes as this is removed, but I'm not sure that's avoidable.